### PR TITLE
Contacts part

### PR DESCRIPTION
**Pull request by RaniArraf**
I have added easy access for the user to both email or whatsapp you. Please let me know if I should add anything.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,17 +52,20 @@ <h2>CONTACTS</h2>
...
-			  	Last name:
+				Rani Arraf:
+			  	<a href = "mailto: arraf.rani@gmail.com">Send Email</a>
 			  	<br>
-			  	<input type="text" name="lastname">
+			 	Rhodeen Davies:
+			  	<a href = "mailto: rhodeendavies@gmail.com">Send Email</a>
 			  	<br>
-			  	
+			  	Llewellyn Devries:
```
Comment (RaniArraf) [inline index.html]: misspelled surname... apologoes

File: index.html [hunk 2]
```diff
@@ -52,24 +52,41 @@ <h2>CONTACTS</h2>
 			<br>
 			<p>All Contact information is found here</p>
```
Comment (rhodeendavies) [inline index.html]: This is kinda redundant, since the users see the "Contact" heading, as well as all the contact info, and if they use the contact button in the landing page, they will know where they're going

File: index.html [hunk 3]
```diff
@@ -52,24 +52,41 @@ <h2>CONTACTS</h2>
...
 			<br>
-			<p>Please enter your information and contact the Area51 Team </p>
+			<p>Here is a list of the whole Area 51 team, please email or whatsapp us if you have a query.</p>
```
Comment (rhodeendavies) [inline index.html]: For the sake of this website, we're the guac team.
Also it's a bit wordy. Remember one of our persona's doesn't like to read, so keep that in mind when designing. Something like "Contact us:" or "The guac team:" will be sufficient

File: index.html [hunk 4]
```diff
@@ -52,24 +52,41 @@ <h2>CONTACTS</h2>
...
 			<form action="contact.html">
-			 	First name:
-			 	<br>
-			 	<input type="text" name="firstname"><br>
-			  	Last name:
+				<!--Gmail logo-->
+				<img src="media/images/gmail.png" alt="mail logo" width="50">
+				<br><br>
+				Rani Arraf:
+			  	<a href = "mailto: arraf.rani@gmail.com">Send Email</a>
```
Comment (rhodeendavies) [inline index.html]: Again, make the wording simpler. Just "Email" or "Message" is enough.

**Comment by RaniArraf** [comment]
I have changed it to look simpler and pulled a request. Thank you!
